### ISSUE TITLE
Using method GetValueOrDefault in NetStandard 2.0 library causes MissingMethodException in netcoreapp2.2 console app when referencing NetStandard library

### ISSUE THREAD

**Issue by Advitalitum** [rsdn/CodeJam#83]
Using method in NetStandard 2.0 library causes MissingMethodException in netcoreapp2.2 when referencing NetStandard library
```csharp
public static TValue GetValueOrDefault<TKey, TValue>(
      [NotNull] this Dictionary<TKey, TValue> dictionary,
      [NotNull] TKey key,
      TValue defaultValue)
```
System.MissingMethodException
  HResult=0x80131513
  Message=Method not found: '!!1 CodeJam.Collections.DictionaryExtensions.GetValueOrDefault(System.Collections.Generic.Dictionary`2<!!0,!!1>, !!0, !!1)'.
  Source=NetStandardLib
  StackTrace:
   at NetStandardLib.NetStandardLibClass.SomeMethod() in C:\dev\CodeJamExceptionRepro\NetStandardLib\NetStandardLibClass.cs:line 10
   at ConsoleApp4.Program.Main(String[] args) in C:\dev\CodeJamExceptionRepro\ConsoleApp4\Program.cs:line 8

Reporo here:
https://github.com/Advitalitum/CodeJamExceptionRepro
OS: Windows 10

**Comment by andrewvk** [comment]
Fixed in #86

**Comment by NN---** [comment]
Shouldn't this issue be closed ?
